components/butterfly/src/server/mod.rs

```diff
@@ -847,10 +847,18 @@ impl Server {
     /// is over 50%, and at least 3 members.
     fn check_quorum(&self, key: &str) -> bool {
         let electorate = self.get_electorate(key);
-
-        let total_population = self.get_total_population(key);
+        let service_group_members = self.get_total_population(key);
+        let total_population = service_group_members.len();
         let alive_population = electorate.len();
 
+        trace!(
+            "check_quorum({}): {}/{} alive/total, electorate: {:?}, service_group: {:?}",
+            key,
+            alive_population,
+            total_population,
+            electorate,
+            service_group_members
```

Comment:
Seems like this could stand to really blow up the logs for very large service groups. Might not be an issue given that you're only going to be enabling `trace` logs in extraordinary circumstances.

Reply:
Yeah, it will totally blow things up, which is why I made it `trace`.